<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GanRaoGuang</class>
 <widget class="QDialog" name="GanRaoGuang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pBGRGhuoqu">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 144, 255);
border-width:2px;
border-style:outset;
border-radius:4px;
color: rgb(255,255,255);
</string>
   </property>
   <property name="text">
    <string>获取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBGRGpingjia">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 144, 255);
border-width:2px;
border-style:outset;
border-radius:4px;
color: rgb(255,255,255);
</string>
   </property>
   <property name="text">
    <string>评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBGetInfo_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 144, 255);
border-width:2px;
border-style:outset;
border-radius:4px;
color: rgb(255,255,255);
</string>
   </property>
   <property name="text">
    <string>清零</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushGRGshushidu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>舒适度评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>干扰光评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBViewStan">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130,130,130);
border-width:2px;
border-style:outset;
border-radius:4px;
color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>查看标准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBViewStan_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130,130,130);
border-width:2px;
border-style:outset;
border-radius:4px;
color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>查看标准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBViewStan_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130,130,130);
border-width:2px;
border-style:outset;
border-radius:4px;
color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>查看标准</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>亮度</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>照度</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>驾驶员干扰光</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cd∕m^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>非熄灯时段</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>熄灯时段</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lux</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>适应亮度</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBCoordinate_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>阈值增量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
